### PR TITLE
Race comparison function

### PR DESCRIPTION
**Pull request by Mari-Hayashi**
The endpoint /compareRace is now working.
I also modified one database file. The database contained 2 lines in which the parser fails to parse correctly. I modified the database so that the parser successfully parses all lines correctly.

### CODE REVIEW

File: src/Server/routes/Analytics.js [hunk 1]
```diff
@@ -51,4 +51,12 @@ router.get('/compareCounties', (req, res) => {
 	});
 });
 
+router.get('/compareRace', (req, res) => {
+	var twoPlacesComperison = AnalyticsModule.GetRaceComparison(req.query.Option);
```
Comment (johan1505) [inline src/Server/routes/Analytics.js]: I think you meant Comparison

**Comment by johan1505** [comment]
Do we want the changes to the .csv files to be pushed to master this time?

File: src/Server/routes/Analytics.js (hunk 1 above)
Comment (Mari-Hayashi) [reply src/Server/routes/Analytics.js]: Fixed.

**Comment by Mari-Hayashi** [comment]
> Do we want the changes to the .csv files to be pushed to master this time?

Yes, as explained in the comment above.

**Pull request merged**